js_modules/dagster-ui/packages/ui-core/src/graph/asyncGraphLayout.ts

```diff
@@ -36,10 +36,32 @@ const asyncGetFullOpLayout = asyncMemoize((ops: ILayoutOp[], opts: LayoutOpGraph
 const _assetLayoutCacheKey = (graphData: GraphData) => {
   // Note: The "show secondary edges" toggle means that we need a cache key that incorporates
   // both the displayed nodes and the displayed edges.
+
+  // Make the cache key deterministic by alphabetically sorting all of the keys since the order
+  // of the keys is not guaranteed to be consistent even when the graph hasn't changed.
+
+  function recreateObjectWithKeysSorted(obj: Record<string, Record<string, boolean>>) {
+    const newObj: Record<string, Record<string, boolean>> = {};
+    Object.keys(obj)
```

Comment:
Not a big deal at all but if you want I think you can use fromEntries to do this and not have to re-declare the type of the object manually on 44 and 55:

```
return Object.fromEntries(Object.entries().sort((a, b) => COMMON_COLLATOR.compare(a[0], b[0])))
```